src/main/kotlin/com/arabyte/arabyteapi/domain/user/service/UserService.kt

```diff
@@ -15,4 +17,9 @@ class UserService(
     fun saveUser(newUser: User): User {
         return userRepository.save(newUser)
     }
+
+    fun getUserOrThrow(userId: Long): User {
```

Comment:
```suggestion
    fun getUser(userId: Long): User {
```
이렇게만 써도 크게 문제 없는게 어차피 return 타입은 User고 문제가 있을 때 throw하는걸 우리가 따로 try catch하지 않고 프론트까지 넘겨주기 때문에 그렇습니다.

Reply:
아하 알겠습니다.